jurity/recommenders/ctr.py

```diff
@@ -14,12 +14,54 @@
 class CTR(_BaseRecommenders):
     """Click-through rate
 
+    Three supported estimation methods:
+
+    1. Matching
     Calculates the CTR using a direct matching method. That is, CTR is only calculated for instances where the
     actual item the user has seen matches the recommendation.
+
+    2. Inverse Propensity Score (IPS)
```

Comment:
lex

Reply:
My thought process here is that doubly robust makes more sense after you've already read about IPS. So the progression is from simplest to most complex calculation (with "matching" first)

Reply:
Yep, I agree with that for the pydoc explanation parts, where we can go not-lex 👍